python/lsst/meas/algorithms/pcaPsfDeterminer.py

```diff
@@ -488,36 +488,36 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
                 if displayExposure:
                     if iterNum > 0:
                         ds9.erase(frame=frame)
-                    maUtils.showPsfSpatialCells(exposure, psfCellSet, self.config.nStarPerCell, showChi2=True,
+                    mautils.showPsfSpatialCells(exposure, psfCellSet, self.config.nStarPerCell, showChi2=True,
                                                 symb="o", size=8, frame=frame,
                                                 ctype=ds9.YELLOW, ctypeBad=ds9.RED, ctypeUnused=ds9.MAGENTA)
                     if self.config.nStarPerCellSpatialFit != self.config.nStarPerCell:
-                        maUtils.showPsfSpatialCells(exposure, psfCellSet, self.config.nStarPerCellSpatialFit,
+                        mautils.showPsfSpatialCells(exposure, psfCellSet, self.config.nStarPerCellSpatialFit,
                                                     symb="o", size=10, frame=frame,
                                                     ctype=ds9.YELLOW, ctypeBad=ds9.RED)
                 if displayResiduals:
                     while True:
                         try:
-                            maUtils.showPsfCandidates(exposure, psfCellSet, psf=psf, frame=4,
+                            mautils.showPsfCandidates(exposure, psfCellSet, psf=psf, frame=4,
                                                       normalize=normalizeResiduals,
                                                       showBadCandidates=showBadCandidates)
-                            maUtils.showPsfCandidates(exposure, psfCellSet, psf=psf, frame=5,
+                            mautils.showPsfCandidates(exposure, psfCellSet, psf=psf, frame=5,
                                                       normalize=normalizeResiduals,
                                                       showBadCandidates=showBadCandidates,
                                                       variance=True)
-                        except:
+                        except Exception:
```

Comment:
I don't think that is reasonable. It's 200 lines of code, with no documentation as to what exceptions it might raise. I admit that many of those lines are contained in their own try/except Exception blocks, but it's still a big job to try to figure out all the possible errors it might raise.